kanidmd/src/lib/valueset.rs

```diff
@@ -1,112 +1,1297 @@
+use crate::credential::Credential;
 use crate::prelude::*;
-// use hashbrown::HashSet;
-use std::borrow::Borrow;
-use std::collections::BTreeSet;
+use crate::repl::cid::Cid;
+use either::Either::{Left, Right};
+use kanidm_proto::v1::Filter as ProtoFilter;
+use smolset::{SmolSet, SmolSetIter};
+use sshkeys::PublicKey as SshPublicKey;
+use std::collections::btree_map::Entry as BTreeEntry;
+use std::collections::{BTreeMap, BTreeSet};
 use std::iter::FromIterator;
+use time::OffsetDateTime;
+use tracing::trace;
+
+use crate::be::dbvalue::{
+    DbCidV1, DbValueCredV1, DbValueEmailAddressV1, DbValueTaggedStringV1, DbValueV1,
+};
+use crate::value::DataValue;
+
+// SmolSet<[; 1]>
+
+#[derive(Debug, Clone)]
+enum I {
```

Comment:
Gonna say no to this one because this variable name is used in a billion Pattern matches and I'm lazy AF to type it out, and it's ONLY a detail of this internal.